cpp/src/io/parquet/reader_impl_preprocess.cu

```diff
@@ -280,6 +280,66 @@ void generate_depth_remappings(std::map<int, std::pair<std::vector<int>, std::ve
   return total_pages;
 }
 
+/**
+ * @brief Helper function to convert an encoding bitmask to a readable string
+ *
+ * @param bitmask Bitmask of found unsupported encodings
+ * @returns Human readable string with unsupported encodings
+ */
+std::string encoding_bitmask_to_str(int8_t encoding_bitmask)
```

Comment:
You might want to loop over the set bits in `encoding_bitmask`, otherwise this won't catch new encodings that might be added to the Parquet spec in the future.